packages/mobile/features/features.ts

```diff
@@ -119,6 +119,18 @@ const features = {
     collectibles: {
         enabled: false,
     },
+    balance: {
+        enabled: false,
+    },
+    profileActions: {
```

Comment:
Lets move these guys inside `wallet`. Also im thinking that `balance` we will always need so perhaps we can remove it